CMakeLists.txt

```diff
@@ -534,7 +547,35 @@ else (DOXYGEN_FOUND)
   message("Doxygen need to be installed to generate the doxygen documentation")
 endif (DOXYGEN_FOUND)
 
-if (SPECFEMPP_ENABLE_PYTHON)
+if (SPECFEMPP_USE_SKBUILD)
     install(TARGETS specfem2d DESTINATION ${SKBUILD_SCRIPTS_DIR})
     install(FILES ${CMAKE_BINARY_DIR}/xmeshfem2D DESTINATION ${SKBUILD_SCRIPTS_DIR} PERMISSIONS OWNER_READ OWNER_WRITE OWNER_EXECUTE)
+endif (SPECFEMPP_USE_SKBUILD)
+
+if (SPECFEMPP_ENABLE_PYTHON)
+    find_package(Python REQUIRED COMPONENTS Interpreter Development.Module)
+    find_package(pybind11 CONFIG REQUIRED)
+    python_add_library(_core MODULE src/specfem2d.cpp WITH_SOABI)
```

Comment:
Is this just taking the bindings from `src/specfem2d.cpp` and then compiling it under `_core`

Reply:
Yes